### PR TITLE
Another option for the play online mini links

### PR DESCRIPTION
**Pull request by alice-blue**
Similar to what I suggested, but this version doesn't require editing any existing messages, I don't think. It adds a separate sentence saying "You can also play online."

### CODE REVIEW

File: www/viewgame [hunk 1]
```diff
@@ -1312,20 +1312,25 @@ if (count($links) == 0 && $dlnotes == "") {
...
+        // Don't show "Play Online" mini link for HTML links; that's what the link itself does
+        // ... unless the HTML is zipped, in which case we *do* need a Play Online link
+        $playOnlineMiniLink = "";
+        if ($playOnlineURL && ($onlineInterpreterType !== "HTML" || $compPri)) {
+            if ($typedesc || $zipnote) {
+                $playOnlineMiniLink = " You can also <a href='$playOnlineURL'>play online</a>.";
+            } else {
+                $playOnlineMiniLink = "<a href='$playOnlineURL'>Play Online</a>";
+            }
+        }
```
Comment (dfabulich) [inline www/viewgame:1330]:
```suggestion
        $playOnlineMiniLink = "";
        if ($playOnlineURL) {
            if ($onlineInterpreterType === "HTML" && !$compPri) {
                // Don't show "Play Online" mini link for HTML links; that's what the link itself does
                // ... unless the HTML is zipped, in which case we *do* need a Play Online link
            } else if ($zipnote || !$typedesc || $onlineInterpreterType === "HTML") {
                $playOnlineMiniLink = " <a href='$playOnlineURL'>Play Online</a>";
            } else {
                $playOnlineMiniLink = " You can also <a href='$playOnlineURL'>play online</a>.";
            }
        }
```

**Review by dfabulich — CHANGES_REQUESTED** [review]
Suggested replacement. This works for tricky cases I tried.

**Comment by dfabulich** [comment]
https://github.com/iftechfoundation/ifdb/pull/1226#issuecomment-2552702002

> I honestly kinda prefer my version here in https://github.com/iftechfoundation/ifdb/pull/1226 better. @salty-horse do you have a preference?

**Comment by salty-horse** [comment]
@dfabulich I prefer the button-next-to-title in the #1226 screenshot, and as a second choice the link-just-below-title.

**Comment by dfabulich** [comment]
Let's go with #1233 instead

**Pull request closed without merging**